src/bin/pg_autoctl/cli_do_root.c

```diff
@@ -244,13 +244,26 @@ keeper_cli_keeper_setup_getopts(int argc, char **argv)
 		{ "group", required_argument, NULL, 'g' },
 		{ "monitor", required_argument, NULL, 'm' },
 		{ "allow-removing-pgdata", no_argument, NULL, 'R' },
-		{ "help", no_argument, NULL, 0 },
+		{ "version", no_argument, NULL, 'V' },
+		{ "verbose", no_argument, NULL, 'v' },
+		{ "quiet", no_argument, NULL, 'q' },
+		{ "help", no_argument, NULL, 'h' },
 		{ NULL, 0, NULL, 0 }
 	};
 
+	/*
+	 * The only command lines that are using keeper_cli_getopt_pgdata are
+	 * terminal ones: they don't accept subcommands. In that case our option
+	 * parsing can happen in any order and we don't need getopt_long to behave
+	 * in a POSIXLY_CORRECT way.
+	 *
+	 * The unsetenv() call allows getopt_long() to reorder arguments for us.
+	 */
+	unsetenv("POSIXLY_CORRECT");
+
 	int optind =
 		cli_create_node_getopts(argc, argv,
-								long_options, "C:D:h:p:l:y:U:A:d:n:f:g:m:R",
+								long_options, "C:D:h:p:l:y:U:A:d:n:f:g:m:RVvqh",
```

Comment:
`H`

Reply:
we will run out of letters soon as we add new parameters. Perhaps we should consider dropping short names for uncommon uses, or better all short options.

Reply:
I have already lost short names and need to come up with new ones  for replication-quorum and candidate priority  options in my PR that is going to be merged after this.

Reply:
Fixed.